usr/src/man/man1m/zfs.1m

```diff
@@ -754,8 +759,11 @@ for volumes is 8 Kbytes. Any power of 2 from 512 bytes to 128 Kbytes is valid.
 This property can also be referred to by its shortened column name,
 .Sy volblock .
 .It Sy written
-The amount of
+The amount of space
 .Sy referenced
+by this dataset, that was written since the previous snapshot
+.Po i.e. that is not referenced by the previous snapshot
```

Comment:
It should. I used the .Po/.Pc when I needed the other macros inside, or when the string length was > 80 characters. You don't really need to "fix" it, just a note.